app/src/main/java/edu/uc/reedws/musiclink/MainActivity.kt

```diff
@@ -15,9 +15,11 @@ import edu.uc.reedws.musiclink.ui.main.ApplicationViewModel
 import edu.uc.reedws.musiclink.ui.main.MainFragment
 import kotlinx.android.synthetic.main.main_activity.*
 
+//Add additional documentation here
+
 class MainActivity() : AppCompatActivity() {
     private lateinit var viewModel: ApplicationViewModel
-
+        //Create the Overlay
```

Comment:
The professor specifically said not to do this

Reply:
Inline comments tend to be avoided by developers.  Code should be self-documenting, and inline comments are often seen as an indication that the code is not self-documenting.  If it is self-documenting, then inline comments are seen as redudant.

Reply:
This and the comment above were just references for myself to find specific lines. They weren't added for any other reason